src/_cffi_src/build_openssl.py

```diff
@@ -110,3 +111,17 @@ def _extra_compile_args(platform):
     libraries=_get_openssl_libraries(sys.platform),
     extra_compile_args=_extra_compile_args(sys.platform),
 )
+
+if __name__ == "__main__":
+    out_dir = os.getenv("OUT_DIR")
```

Comment:
Does the setuptools build machinery set this value?

Reply:
No, the rust build machinery (from which this is called) does